public/bibliography.html

```diff
@@ -79,10 +82,10 @@
       </nav>
    </header>
 
-   <div class="container">
+   <div class="container" id="main-content">
       <h1 class="page-title hyphenate">BLE LTER Publications</h1>
       <br>
-      <p class="lead">Search our bibliography, or view <a href="https://www.zotero.org/groups/2211939/lter-ble/items" target="_blank" rel="noopener" title="BLE Zotero Group">our library on Zotero</a>.
+      <p class="lead">Search our bibliography, or view <a href="https://www.zotero.org/groups/2211939/lter-ble/items" target="_blank" rel="noopener" title="BLE Zotero Group" aria-label="open in new tab">our library on Zotero</a>.
```

Comment:
shouldn't this say, "open BLE Zotero library in new tab"?

Reply:
I was doing link-specific labels but then got tired and started doing generic "open in new tab" labels. We can go back and make it specific again.

Reply:
specific seems like the intent of the rule to me

Reply:
more specifics added 8b4d5770e0d587dcdfe0e232bbfd18a1bce3c889

Reply:
[This site](https://www.deque.com/blog/text-links-practices-screen-readers/) has a The Comparison section which shows how screen readers read such links. Look at row "C. Aria- label present besides anchor text".  Sometimes they read anchor text, sometimes label, sometimes both.  I suppose in this case redundancy is better than an ambiguous link, so I think the way you have it in 8b4d577 is good.